drivers/builtin/src/entropy_poll.h

```diff
@@ -57,6 +41,35 @@ int mbedtls_nv_seed_poll(void *data,
                          unsigned char *output, size_t len, size_t *olen);
 #endif
 
+/* Wrapper to allow mbedtls_platform_get_entropy_alt() to be used from entropy
+ * module as other entropy polling functions (i.e. mbedtls_platform_entropy_poll
+ * and mbedtls_nv_seed_poll). */
+#if defined(MBEDTLS_PLATFORM_GET_ENTROPY_ALT)
+
+#include <mbedtls/platform.h>
+
+#define MBEDTLS_ENTROPY_MIN_HARDWARE            32
+
+static inline int mbedtls_hardware_poll(void *data, unsigned char *output,
```

Comment:
Why is this an inline function in a header file? It's only called inside `entropy.c`, so why not define it there and make it `MBEDTLS_STATIC_TESTABLE`?

Reply:
Actually this function is also called from `test_suite_psa_crypto_init.function` in `custom_entropy_init()` function.  If we want to remove the `inline` part, I can move the implementation to `entropy.c` and keep only the definition here. How does this sound?

Reply:
`MBEDTLS_STATIC_TESTABLE` lets you call a function from the tests as long as the tests are guarded by `MBEDTLS_TEST_HOOKS`.